packages/blade/src/components/Input/OTPInput/OTPInput.stories.tsx

```diff
@@ -181,7 +202,12 @@ export default {
 } as Meta<OTPInputProps>;
 
 const OTPInputTemplate: StoryFn<typeof OTPInputComponent> = ({ ...args }) => {
-  return <OTPInputComponent {...args} />;
+  const maxWidth = args.otpLength === 4 ? '376px' : '568px';
```

Comment:
So the OTPInput itself doesn't have MaxWidth is it?

Reply:
Can't find it in ts so guessing nahi hai. We can add it if someone raises it though